### DISCUSSION TITLE
iot network

### DISCUSSION THREAD

**Discussion by TenKoalaKing** [maxbundscherer/independer-loras#98] [Ideas]
Which iot network is this going to use? I think it should be helium but I also want to know when these things will be available.

**Comment by maxbundscherer** [comment]
We are not using any existing IoT infrastructure or protocol. The [Independer-Protocol](https://github.com/maxbundscherer/independer-loras/blob/master/docs/independer-protocol.md) is based on the [LoRaS-Protocol](https://github.com/maxbundscherer/independer-loras/blob/master/docs/loras-protocol.md).

@TenKoalaKing - What advantages would be if we used a protocol like helium?

The software is already working, and the case's 3D printing [files](https://github.com/maxbundscherer/independer-loras/tree/master/case) are also available. The current state of development and from the first version can be seen on this [page](https://github.com/maxbundscherer/independer-loras/projects/2).
